public/app/features/provisioning/Wizard/ConnectPage.tsx

```diff
@@ -1,18 +1,27 @@
+import { useParams } from 'react-router-dom-v5-compat';
+
 import { Page } from 'app/core/components/Page/Page';
 
 import { ProvisioningWizard } from './ProvisioningWizard';
+import { RepoType } from './types';
 
 export default function ConnectPage() {
+  const { type } = useParams<{ type: RepoType }>();
+
+  if (!type) {
+    return null;
+  }
+
   return (
     <Page
       navId="provisioning"
       pageNav={{
-        text: 'Connect to external storage',
+        text: type === 'github' ? 'Configure Git Sync' : 'Configure local file path',
```

Comment:
🤣 this won't take long to bite us in the butt... but good enough for now

Reply:
A compromise on the naming :) But we'd remove it after G12.